hw03-[1,2,3].ipynb

```diff
@@ -0,0 +1,127 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from mrjob.job import MRJob\n",
+    "import csv\n",
+    "import statistics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Overwriting hw03-1.py\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%file hw03-1.py\n",
+    "class avgwords (MRJob):\n",
+    "    def Mapper (self,_,line):\n",
```

Comment:
mrjob needs class methods called `.mapper()` and `.reducer()`, not `.Mapper()` and `.Reducer()`:
```
Traceback (most recent call last):
  File "hw3.py", line 17, in <module>
    avgwords.run()
  File "./python3.8/site-packages/mrjob/job.py", line 616, in run
    cls().execute()
  File "./python3.8/site-packages/mrjob/job.py", line 687, in execute
    self.run_job()
  File "./python3.8/site-packages/mrjob/job.py", line 636, in run_job
    runner.run()
  File "./python3.8/site-packages/mrjob/runner.py", line 495, in run
    raise ValueError('Job has no steps!')
ValueError: Job has no steps!
```
-3